python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1621,6 +1621,9 @@ def _empty_like(self, keep_index=True):
         )
 
     def _split(self, splits, keep_index=True):
+        if len(self._index) == 0:
```

Comment:
```suggestion
        if self._num_rows == 0:
```

This can be just `self._num_rows`